### DISCUSSION TITLE
Can I only use pb.collections.getList() if I am authenticated as an admin?

### DISCUSSION THREAD

**Discussion by j2m6lz5n** [pocketbase/pocketbase#4294] [Q&A] [answered]
I want to return a list of all of the names of each collection in the db. My goal is to populate a <ul> filled with dynamically generated links based on the names of the collections.

I run this:  `pb.collections.getList(page = 1, perPage = 30, options = {}); `
But I get back: 
`'The request requires valid admin authorization token to be set.' `

From the sdk I can see this: 
`// Returns a paginated collections list. 🔐pb.collections.getList(page = 1, perPage = 30, options = {});` 
Which I guess means it can only be run by an admin. And it certainly works as admin.  
Is there a different or accepted way to get all of the collection names to serve in my app or do I need to login as admin?

I don't know if this is a reasonable concern, but I am afraid that logging in as an admin for every request made by any non logged in user is a bad idea. 

Thank you in advance for any help that you might offer.

**Comment by ganigeorgiev** [comment, marked as answer]
The [Collections web APIs](https://pocketbase.io/docs/api-collections/) require admin authentication.

I don't understand your use case nor your concerns, but if you want just the collection names, an alternative could to be create a helper [View collection](https://pocketbase.io/docs/collections/#view-collection) that will have its own API rules and that you can make public. An example query could be:

```sql
SELECT id, name
FROM _collections
```

**Reply by j2m6lz5n** [reply]
Thank you for your  prompt response.
